src/sentry/models/groupsubscription.py

```diff
@@ -178,7 +178,8 @@ class GroupSubscription(Model):
 
     project = FlexibleForeignKey("sentry.Project", related_name="subscription_set")
     group = FlexibleForeignKey("sentry.Group", related_name="subscription_set")
-    user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, on_delete="CASCADE")
+    user_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="CASCADE")
+    team_id = FlexibleForeignKey("sentry.Team", null=True, db_index=True, on_delete=models.CASCADE)
```

Comment:
```suggestion
    team = FlexibleForeignKey("sentry.Team", null=True, db_index=True, on_delete=models.CASCADE)
```

Relations should use names with out `_id`. Including `_id` in the field is resulting in a column named `team_id_id`.